README.md

```diff
@@ -1 +1,297 @@
+The filter button gives a filtring on the specified group (the name of the button), with all their connecting nodes.
+An example of such an filter is shown in the following image.
+
+![filter](images/filter.png)
+
+The legend is based on the visible nodes and links. When diffrent nodes or links are added, the legend will be updated.
+
+### Interactions
+
+#### Hide Groups
+
+When clicking a group of the legend, this group will disappear. When clicking again, it reverts to previous.
+
+![hide group](images/hide_group.png)
+
+#### Hide Links
+
+When clicking a link of the legend, this link will disappear. When clicking again, it reverts to previous.
+
+![hide group](images/hide_link.png)
+
+
+#### Selecting a Node
+
+When clicking a node, this node will be selected and the info will be visible in the top left corner.
+
+![select node](images/select_node.png)
+
+The image shows the info of the selected node. There are two more buttons showed as well. These two buttons pop up when a node is selected. These will be explained in [highlight connecting nodes](####highlight_connecting_nodes) and [Add/show connecting nodes](####add/show_connecting_nodes).
+
+#### Hovering a Node
+
+When hovering a nodes, the name of that node is shown.
+In the previous image, the black node at the bottom was hovered by the mouse.
+
+#### HTML link in Info of Node
+
+When `html_documentation_root` is specified, the name of the node will be a link to the documentation of that specific node.
+
+#### Highlight Connected Nodes
+
+When the button with the eye is clicked, the nodes connecting to the selected node will be highlighted. The rest of the nodes get a lower opacity.
+
+![highlight connected nodes](images/highlight_connected_nodes.png)
+
+
+#### Add/Show Connecting Nodes
+
+When the button **Add/show connecting nodes** is clicked, the nodes connecting to the selected node will be obtained. It will add new nodes or links, if there are any.
+
+Before pushing the button:
+![before adding](images/before_adding.png)
+
+After pushing the button:
+![add connecting nodes](images/add_connecting_nodes.png)
+
+#### Node labels
+
+The labels/names of the nodes are shown when the checkbox of **Nodes with labels** is active.
+
+![show labels](images/show_labels.png)
+
+#### Zoom buttons
+
+There are three buttons for zooming, respectively zoom in, zoom out and zoom till the network fits the canvas.
+
+![zoom buttons](images/zoom.png)
+
+#### Errors
+
+When an error occurs you get a notification at the top of the page as shown in following example where an invalid Cypher query is submitted:
+
+![error](images/error.png)
+
+You can show the error by clicking **Show error message**:
+
+![error](images/show_error.png)
+
+You can close the error by clicking **X**.
+If the error is not closed and another error occurs, all errors will be listed at the top of the page.
+You will be redirected to the hight of the newest error.
```

Comment:
```suggestion
If the error is not closed and another error occurs, the new error pops up at the top.
```

Reply:
Suggestion applied in commit 7ee06db64ba6d1a14607ea363ebe43ae66775260